examples/seismic/viscoacoustic/operators.py

```diff
@@ -155,35 +155,32 @@ def sls_2nd_order(model, geometry, p, r=None, **kwargs):
     # Density
     rho = 1. / b
 
-    # Bulk modulus
-    bm = rho * vp**2
-
     # Attenuation Memory variable.
     r = r or TimeFunction(name="r", grid=model.grid, time_order=2,
                           space_order=space_order, save=save_t, staggered=NODE)
 
     if forward:
+        # Attenuation Memory variable.
```

Comment:
What's a memory variable btw?

Reply:
It's just how it's called it's the extra PDE that model the attenuation